jni/rubicon.c

```diff
@@ -825,8 +825,9 @@ jobjectRefType GetObjectRefType(jobject obj) {
 JNIEXPORT jint JNICALL Java_org_beeware_rubicon_Python_init(JNIEnv *env, jobject thisObj, jstring pythonHome, jstring pythonPath, jstring rubiconLib) {
     int ret = 0;
     char pythonPathVar[512];
+#ifdef ANDROID
```

Comment:
I have some commentary (not an objection):

Nitpick, but I propose you move this variable declaration into the `ifdef` below where we use it. Not a showstopper.

I also am not sure we're even using this on Android. I ran out of time tonight to refactor this, but I am hopeful that I will refactor this variable away anyway.

Reply:
Fixed as suggested, pending your refactor if needed when you get a chance.